README.md

```diff
@@ -1,49 +1,185 @@
-# Asciidoctor.js PDF
+# Asciidoctor PDF
 
 [![Build Status](https://travis-ci.org/Mogztter/asciidoctor-pdf.js.svg?branch=master)](https://travis-ci.org/Mogztter/asciidoctor-pdf.js)
 [![npm version](https://img.shields.io/npm/v/asciidoctor-pdf.svg)](https://www.npmjs.org/package/asciidoctor-pdf)
 
-Convert AsciiDoc document to PDF using Asciidoctor.js and Puppeteer (Headless Chrome)
+A PDF converter for AsciiDoc based on web technologies.
+It allows complex layouts to be defined with CSS and JavaScript, while writing the content in AsciiDoc.
 
-## About
+Asciidoctor PDF has support for LaTeX-style mathematical equations (via [MathJax](https://www.mathjax.org/)) and syntax highlighting (via [highlight.js](https://highlightjs.org/)).
+Many more features can be added simply by importing an existing JavaScript or CSS framework.
```

Comment:
I try to avoid "simple/simply" when explaining something new to someone else as it makes a lot of assumptions. "Simple" in this case assumes that you the recipient is familiar with JavaScript and CSS, among probably other things like npm. I suggest keeping the sentence, but removing "simply"

Reply:
You're absolutely right!

```suggestion
Many more features can be added by importing an existing JavaScript or CSS framework.
```